src/App.js

```diff
@@ -0,0 +1,52 @@
+import InputView from './views/InputView.js';
+import { Validator } from './models/Validator.js';
+import OutputView from './views/OutputView.js';
+import ScoreBoard from './models/ScoreBoard.js';
+import Game from './models/Game.js';
+
+class App {
+	#scoreBoard;
```

Comment:
혹시 editor 상의 인덴트 설정이 tab으로 설정되어 있으실까요? 프론트엔드 개발에서는 SPACE 2칸이 국룰(?)이라 변경해서 사용해주시면 좋을 것 같아요 :)

```suggestion
  #scoreBoard;
```

Reply:
짚어주셔서 감사합니다!

수정하겠습니다 :)